src/styles/types.js

```diff
@@ -218,6 +218,7 @@ export type SizingT = {
   scale300: string,
   scale400: string,
   scale500: string,
+  scale550?: string,
```

Comment:
Do we use `scale550` anywhere? For the official set of components we should stick to the 4px baseline grid.

Reply:
Used 600 like you suggested.